test/parallel/test-fs-access.js

```diff
@@ -97,7 +97,8 @@ fs.promises.access(readOnlyFile, fs.constants.R_OK)
   };
   const expectedErrorPromise = (err) => {
     expectedError(err);
-    assert.match(err.stack, /at async Object\.access/);
+    assert.strictEqual(err.code, 'ENOENT');
```

Comment:
Does the error code changed or just the stacktrace?

Reply:
the stack trace changed.